orange-datamodel/src/test/java/com/hartwig/hmftools/datamodel/OrangeJsonTest.java

```diff
@@ -217,8 +217,8 @@ private static void assertPurple(@NotNull PurpleRecord purple) {
         assertEquals("ENST00000591126", gainLoss.transcript());
         assertFalse(gainLoss.isCanonical());
         assertEquals(CopyNumberInterpretation.FULL_LOSS, gainLoss.interpretation());
-        assertEquals(0, gainLoss.minCopies());
-        assertEquals(1, gainLoss.maxCopies());
+        assertEquals(0.1, gainLoss.minCopies(), 0.001);
+        assertEquals(1.2, gainLoss.maxCopies(), 0.001);
```

Comment:
Specific reason why you use 0.001 rather than reuse `EPSILON`?

Reply:
Fixed in all places